manual.md

```diff
@@ -118,4 +122,14 @@ The Optimism Foundation will undertake this ministerial work with a view towards
 
 ### **Amendments**
 
-The procedures described in this OPerating Manual will go into effect at the time of Airdrop #1, and be adjusted later in connection with a series of governance experiments. These changes include but are not limited to introducing the Citizens’ House, and adding, removing, and modifying Proposal Types and rules relating to voting processes. Any non-clerical updates to the OPerating Manual will be reflected with a new version number at the top of this document, at which point the updated version will go into effect.
+The procedures described in this Operating Manual will go into effect as releases are published on GitHub. Major releases to the manual will be made in connection with a series of governance experiments (“Seasons”). These changes include but are not limited to introducing the Citizens’ House, and adding, removing, and modifying Proposal Types and rules relating to voting processes. Any non-clerical updates to the Operating Manual will be reflected with a new version number at the top of this document, at which point the updated version will go into effect.
+
+## Process TLDR
```

Comment:
This should probably be at the top of the doc, with links to document sections that go into more depths. Maybe like a table of contents/TL;DR mix?